src/app/utils/tests/TestUtils.spec.ts

```diff
@@ -220,39 +238,58 @@ export class SimpleComponentTestUtils<T> {
         const elementClasses: string[] = Utils.getNonNullable(element.attributes.class).split(' ').sort();
         expect(elementClasses).withContext(`For ${elementName}`).toEqual(classesSorted);
     }
+
     public expectElementNotToExist(elementName: string): void {
         const element: DebugElement = this.findElement(elementName);
         expect(element).withContext(`${elementName} should not exist`).toBeNull();
     }
+
     public expectElementToExist(elementName: string): DebugElement {
         const element: DebugElement = this.findElement(elementName);
         expect(element).withContext(`${elementName} should exist`).toBeTruthy();
         return element;
     }
+
     public expectElementToBeEnabled(elementName: string): void {
         const element: DebugElement = this.findElement(elementName);
         expect(element).withContext(`${elementName} should exist`).toBeTruthy();
         expect(element.nativeElement.disabled).withContext(elementName + ' should be enabled').toBeFalsy();
     }
+
     public expectElementToBeDisabled(elementName: string): void {
         const element: DebugElement = this.findElement(elementName);
         expect(element).withContext(`${elementName} should exist`).toBeTruthy();
         expect(element.nativeElement.disabled).withContext(`${elementName} should be disabled`).toBeTruthy();
     }
+
     public expectTextToBe(elementName: string, expectedText: string): void {
         const element: DebugElement = this.findElement(elementName);
         expect(element).withContext(`${elementName} should exist`).toBeTruthy();
         expect(element.nativeNode.innerHTML).toEqual(expectedText);
     }
+
     public fillInput(elementName: string, value: string): void {
         const element: DebugElement = this.findElement(elementName);
         expect(element).withContext(`${elementName} should exist in order to fill its value`).toBeTruthy();
         element.nativeElement.value = value;
         element.nativeElement.dispatchEvent(new Event('input'));
     }
+
+    public async selectChildElementOfDropDown(dropDownName: string, childName: string): Promise<void> {
+        const selectedDropDOwn: HTMLSelectElement = this.findElement(dropDownName).nativeElement;
+        // eslint-disable-next-line guard-for-in
+        for (const o in selectedDropDOwn.options) {
+            console.log(dropDownName, 'has', o);
+        }
```

Comment:
as-tu lu la description de la règle désactivée, et peux-tu expliquer pourquoi ça fait sens de la désactiver ici plutôt que de checker le `hasOwn` comme ils expliquent ? https://eslint.org/docs/latest/rules/guard-for-in

Reply:
J'n'avais pas lu, là j'ai survolé rapidement, mais tpr